### PR TITLE
Corrections to image-classification.md

### PR DESCRIPTION
**Pull request by mstfbl**
Corrected the ".tsv" file name to "tags.tsv".
Corrected the method name PairAndDisplayResults() to DisplayResults() as PairAndDisplayResults() was renamed to DisplayResults().

[AB#1596621](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1596621)

### CODE REVIEW

**Comment by Youssef1313** [comment]
It's not a file name, this is the snippet identifier. I think it should remain tsv.

File: docs/machine-learning/tutorials/image-classification.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Your layer uses a [multinomial logistic regression algorithm](https://en.wikiped
 There are two data sources: the `.tsv` file, and the image files.  The `tags.tsv` file contains two columns: the first one is defined as `ImagePath` and the second one is the `Label` corresponding to the image. The following example file doesn't have a header row, and looks like this:
 
 <!-- markdownlint-disable MD010 -->
-```tsv
+```tags.tsv
```
Comment (natke) [inline docs/machine-learning/tutorials/image-classification.md]:
```suggestion
```tsv
```

**Review by natke — CHANGES_REQUESTED** [review]

**Comment by natke** [comment]
Thank you @mstfbl/ I've requested one small change.

**Comment by natke** [comment]
Will fix as part of AB#1596621

**Pull request closed without merging**